README.md

```diff
@@ -59,7 +59,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 Before you contribute, please read through the contributing and developer guides to get an idea of what kinds of pull requests we will or won't accept.
 
 * [Contributing Guide](documentation/wiki/Contributing-Code.md)
-* [Developer Guide](documentation/wiki/Building-Testing-and-Debugging.md)
+* [Developer Guide](documentation/wiki/Building-Testing-and-Debugging-on-Full-Framework-MSBuild.md)
```

Comment:
Could you also add a line for the .net core page? (I'd skip mono as that information is really stale)